docs/content/guide/installation/index.md

```diff
@@ -59,6 +59,30 @@ docker run \
     latest
 ```
 
+**Notes**
+1. In order to complete prompts with docker you will need to use an [interactive terminal](https://docs.docker.com/engine/reference/commandline/run/#assign-name-and-allocate-pseudo-tty---name--it)
+1. You may also want to include your own user and group ID if any files would be created using the [user option](https://docs.docker.com/engine/reference/run/#user).
+
+```sh
+docker run \
+    --mount type=bind,source=$PWD,target=/src \
+    -w /src \
+    -it \
+    --user $(id -u ${USER}):$(id -g ${USER}) \
+    ghcr.io/miniscruff/changie \
+    new
+
+```sh
+docker run \
+    --mount type=bind,source=$PWD,target=/src \
+    -w /src \
+    -it \
+    ghcr.io/miniscruff/changie \
+    new
+```
+
+**NOTE** : The `it` option is not required in batch mode when you specify all required flags for the `new` command.
```

Comment:
Should only need one example now, and this note is handled by the first note above.
```suggestion
```
```

Be sure to close the code block above though.

Reply:
Done !